src/Assignment-4/assignment4.html

```diff
@@ -6,13 +6,72 @@
 	<meta http-equiv="Pragma" content="no-cache" />
 	<meta http-equiv="Expires" content="0" />
 	<script src="bundle.js" type="text/javascript"></script>
+	<link rel="stylesheet" type="text/css" href="./reset.css">
+	<link rel="stylesheet" type="text/css" href="./assignment4.css">
+	<link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
 </head>
 
 <body>
 	<div id="container">
 
+
 	<!-- Start your code from here -->
-	
+	<header>
+		
+		<img src ="\html-css-latest\src\Assignment-4\images\logo.png" alt ="Dog-logo">
+		<nav>
+			<ul>
+				<li><a href ="#" >Home</a></li>
+				<li><a href ="#" >Things</a></li>
+				<li><a href ="#" >Stuff</a></li>
+			</ul>
+		</nav>
+	</header>
+
+	<section id ="hero">
+		<h2>A Site About Some Dogs</h2>
+		<img src ="\html-css-latest\src\Assignment-4\images\inside round back ground.jpg" alt="back-ground">
```

Comment:
Try to keep image names short and a one word. You can use underscore or hyphen while naming.